assets/global.js

```diff
@@ -553,6 +564,131 @@ class SliderComponent extends HTMLElement {
 
 customElements.define('slider-component', SliderComponent);
 
+class SlideshowComponent extends SliderComponent {
+  constructor() {
+    super();
+    this.sliderControlWrapper = this.querySelector('.slider-buttons');
+    this.bannerContents = this.slider.querySelectorAll('.banner__content');
+
+    if (!this.sliderControlWrapper) return;
+
+    this.sliderItemsToShow = Array.from(this.sliderItems).filter(element => element.clientWidth > 0);
+    this.sliderLastItem = this.sliderItemsToShow[this.sliderItemsToShow.length - 1];
+    if( this.sliderItemsToShow.length > 0) this.currentPage = 1;
```

Comment:
Isn't that `this.currentPage = 1;` automatically inherited from `SliderComponent` initialization? Or are there scenarios where this could be initialized with a different value?

Reply:
I thought I needed it because it comes up as undefined when I console log it on initialization. But seems to work fine without.